client/my-sites/checkout/checkout-thank-you/marketplace/use-default-thank-you-footer.tsx

```diff
@@ -26,45 +26,41 @@ export function useDefaultThankYouFoooter( slugs: string[] ): ThankYouSectionPro
 		nextStepsClassName: 'thank-you__footer',
 		nextSteps: [
 			{
-				stepKey: 'thank_you_footer_support_guides',
-				stepTitle: translate( 'Support guides' ),
+				stepKey: 'thank_you_footer_explore',
+				stepTitle: translate( 'Keep growing' ),
 				stepDescription: translate(
-					'Our guides will show you everything you need to know about plugins.'
+					'Take your site to the next level. We have all the solutions to help you.'
 				),
 				stepCta: (
 					<Button
 						isLink
-						href="https://wordpress.com/support/plugins/"
+						href={ `/plugins/${ siteSlug }` }
 						target="_blank"
```

Comment:
Wondering if it would be better keeping the user in the same window for this specific link 🤔